assets/panel/debugger.properties

```diff
@@ -542,6 +542,14 @@ scopes.notAvailable=Scopes unavailable
 # for when the debugger is not paused.
 scopes.notPaused=Not paused
 
+# LOCALIZATION NOTE (scopes.toggleToGenerated): Scopes right sidebar pane label
+# for link to change scope list to show scopes for generated code.
```

Comment:
I think both comments need some rewording, but I'm not sure I can help a lot, not being familiar with the matter. E.g. "Link displayed in the right sidebar to display scopes for generated code" makes sense?

Reply:
Haha, yeah that's what I get for copy/pasting half the text from the one above this :P